api/app/src/external/commonwell/link/create.ts

```diff
@@ -30,37 +32,41 @@ export const create = async (
 
     if (cwPersonId) {
       await reset(patient, organization);
-
-      await setCommonwellId({
-        patientId: patient.id,
-        cxId: patient.cxId,
-        commonwellPatientId: cwPatientId,
-        commonwellPersonId: undefined,
-      });
     }
 
     const orgName = organization.data.name;
     const orgId = organization.id;
     const commonWell = makeCommonWellAPI(orgName, oid(orgId));
```

Comment:
General comment:

This all looks good, however creating a mock CW object could be cleaner. With this approach, we need to modify the logic flow everywhere we talk to CW -> this could be more error prone, and more difficult to maintain, since we'll always have to think about sandbox when developing.

What if instead we change the logic in  `makeCommonWellAPI()` to return a CommonWellMock object if the API is running in Sandbox? This way, we don't have to change the flow of logic in the API at all, and can just mock the returns of the CW API.

Ideally, we'd refactor this (and actually we should probably make this a pattern) to have `CommonWell()` be an interface, not an actual object, and have classes that implement the interface to either be the actual API, or the mock. This way, we can just always return `CommonWell()` from `makeCommonWellAPI()`, and won't have to return 2 types, ie `CommonWell | CommonWellMock`.

Lmk what you think - happy to chat about this